### PR TITLE
game-03: Copy moddable-platformer to game-03

### PR DESCRIPTION
**Pull request by starnight**
Copy moddable-platformer to game-03, but detach GDscripts basically.
However, to make player move and collect coins, add some Block Coding
blocks to implement the behaviors. Besides, the player uses A & D keys
to walk left & right and W key for jumping.

This is not a beginning step as the baseline. More features will come
later.

https://phabricator.endlessm.com/T35632

### CODE REVIEW

File: game-03/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Godot Minigames
```
Comment (manuq) [inline game-03/README.md]: Can you remove the docs from the moddable platformer? Instead use this template:
```
# [game title]

## About
[What is this game about? Describe the goal and the challanges.]

## Implementation
[Please describe how you managed to implement this without a single line of GDScript. Which blocks were useful? Which editor features and nodes? Have you found interesting ways of using the current blocks? Have you been surprised by something that required no programming at all in Godot?]

## Limitations found
[What had to be workarounded and should be easier? What wasn't possible? Which blocks were you missing?]

```

File: game-03/doc/MODS.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Available modifications
```
Comment (manuq) [inline game-03/doc/MODS.md]: Please remove this file.

**Review by manuq — CHANGES_REQUESTED** [review]
Nice start!

For keeping the demo simple, please remove the modding assets. Leave a single "skin" for the level and for the player.

Some observations from playing:
- The enemies don't hurt the player and they stay in the same position. Is this work in progress? Or was it a limitation you found in the block coding?
- The HUD looks too big and the label "Press any key..." is cut. This project has a window size that is 1/2 of the moddable platformer. So you can probably scale it.
- The label "Press any key..." stays after I press a key that's not Enter (UI Accept). So please change the text to "Press Enter to start" or remove it.
- The camera has too much zoom, so the platforms around can't be seen before jumping.
- There is an issue with the collision between the player and the platform. Sometimes the legs are inside the platform.

**Comment by manuq** [comment]
One more thing: You added a "move and slide" block below "move with player buttons". But the "move with player buttons" is already calling move_and_slide(). Check `addons/block_code/blocks/input/characterbody2d_move.tres`.

File: game-03/README.md (hunk 1 above)
Comment (starnight) [reply game-03/README.md]: sure.  Delete it first.  Will add the README.md back after the game development reaches a milestone.

File: game-03/doc/MODS.md (hunk 2 above)
Comment (starnight) [reply game-03/doc/MODS.md]: ok

**Review by manuq — APPROVED** [review]

**Pull request merged**